src/libraries/System.Security.Cryptography.Encoding/tests/Cbor/CborWriter.Map.cs

```diff
@@ -17,49 +17,24 @@ public partial class CborWriter
 
         /// <summary>
-        ///   Writes the start of a definite-length map (major type 5).
+        ///   Writes the start of a map (major type 5).
         /// </summary>
-        /// <param name="definiteLength">The definite length of the map.</param>
+        /// <param name="definiteLength">
+        ///   Writes a definite-length map if inhabited,
+        ///   or an indefinite-length map if <see langword="null" />.
+        /// </param>
         /// <exception cref="ArgumentOutOfRangeException">
         ///   The <paramref name="definiteLength"/> parameter cannot be negative.
         /// </exception>
         /// <exception cref="InvalidOperationException">
         ///   Writing a new value exceeds the definite length of the parent data item. -or-
-        ///   The major type of the encoded value is not permitted in the parent data item
-        /// </exception>
-        /// <remarks>
-        ///   Map contents are written as if arrays twice the length of the map's declared size.
-        ///   For instance, a map of size <c>1</c> containing a key of type int with a value of type string
-        ///   must be written by successive calls to <see cref="WriteInt32(int)"/> and <see cref="WriteTextString(ReadOnlySpan{char})"/>.
-        ///   It is up to the caller to keep track of whether the next call is a key or a value.
-        ///
-        ///   Fundamentally, this is a technical restriction stemming from the fact that CBOR allows keys of any type,
-        ///   for instance a map can contain keys that are maps themselves.
-        /// </remarks>
-        public void WriteStartMap(int definiteLength)
-        {
-            if (definiteLength < 0 || definiteLength > int.MaxValue / 2)
-            {
-                throw new ArgumentOutOfRangeException(nameof(definiteLength));
-            }
-
-            WriteUnsignedInteger(CborMajorType.Map, (ulong)definiteLength);
-            PushDataItem(CborMajorType.Map, definiteLength: 2 * definiteLength);
-            _currentKeyOffset = _offset;
-        }
-
-        /// <summary>
-        ///   Writes the start of an indefinite-length map (major type 5).
-        /// </summary>
-        /// <exception cref="InvalidOperationException">
-        ///   Writing a new value exceeds the definite length of the parent data item. -or-
         ///   The major type of the encoded value is not permitted in the parent data item. -or-
-        ///   The written data is not accepted under the current conformance level
+        ///   The written data is not accepted under the current conformance mode.
         /// </exception>
         /// <remarks>
-        ///   In canonical conformance levels, the writer will reject indefinite-length writes unless
+        ///   In canonical conformance modes, the writer will reject indefinite-length writes unless
         ///   the <see cref="ConvertIndefiniteLengthEncodings"/> flag is enabled.
-        ///
+        ///   
```

Comment:
I'm surprised that this trailing whitespace isn't provoking an analyzer error.

Also:
I don't remember how the doc comments to doc site importer handles paragraphs here, but canonically if you want this remarks to be in 3 different paragraphs you should wrap each of them in a `<para>...</para>` element.

Reply:
I think it's because the files are still in the test project.